Settings.c

```diff
@@ -356,11 +356,11 @@ Settings* Settings_new(int cpuCount) {
 
    this->sortKey = PERCENT_CPU;
    this->direction = 1;
-   this->hideThreads = false;
+   this->hideThreads = true;
```

Comment:
This is a different setting actually. Just checked in the source. It's referenced at https://github.com/htop-dev/htop/blob/master/linux/LinuxProcessList.c#L865-L867 to implement some RedHat magic …

Reply:
The SHIFT+H keycombo changes both hideThreads nad hideUserlandThreads, I wasn't entirely sure what it did, but wanted it to be consistent with the shortcut.

Should I then only change hideUserLandThreads to true?

Reply:
Good question. Maybe see what the devs have to say about this.